rust/src/lib.rs

```diff
@@ -17,8 +19,13 @@ fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
 
 /// The entry point for defining the Python module.
 #[pymodule]
-fn synapse_rust(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+fn synapse_rust(py: Python<'_>, m: &PyModule) -> PyResult<()> {
+    pyo3_log::init();
+
     m.add_function(wrap_pyfunction!(sum_as_string, m)?)?;
```

Comment:
Should we remove the `sum_as_string` at some point or are you leaving that as an easy to check that things work?

Reply:
Yeah, I think we should replace it with something else to make it easier to test Rust. That is another PR though.

Reply:
Yes, unrelated -- sorry I didn't make that clear! Just something I noticed.